agentops/llm_tracker.py

```diff
@@ -27,6 +26,9 @@ class LlmTracker:
         "cohere": {
             "5.4.0": ("chat", "chat_stream"),
         },
+        "ollama": {
```

Comment:
ollama support was added in another PR! can you revert this file? we'd rather keep the comments in for now